### DISCUSSION TITLE
Nova v5 Support

### DISCUSSION THREAD

**Discussion by nw-b** [spatie/nova-tags-field#184] [Ideas]
Nova 5 has dropped! This package needs to be updated to support the new version.
https://nova.laravel.com/docs/v5/releases

```
  Problem 3
    - Root composer.json requires spatie/nova-tags-field ^4.0 -> satisfiable by spatie/nova-tags-field[4.0.0, ..., 4.0.8].
    - spatie/nova-tags-field[4.0.0, ..., 4.0.3] require laravel/nova ^4.0 -> found laravel/nova[4.0.0, ..., 4.35.5] but it conflicts with your root composer.json require (^5.0).
    - spatie/nova-tags-field[4.0.4, ..., 4.0.8] require laravel/nova ^4.13 -> found laravel/nova[4.13.0, ..., 4.35.5] but it conflicts with your root composer.json require (^5.0).
```

**Comment by freekmurze** [comment]
I'd accept a PR that makes this package compatible with Nova 5

**Reply by crynobone** [reply]
@freekmurze can you make a separate branch to accept such PR? Nova 5 contains multiple breaking changes and it would be hard to allow both Nova 4 and 5.

**Reply by kichetof** [reply]
Hi guys, latest version 5.0.1 with @crynobone changes https://github.com/spatie/nova-tags-field/pull/187 is broken _(tested with Nova 5.7.7 only)_. Without any changes, nothing is displayed in both index, detail and form views. Revert to 5.0.0 fix the issue. Any ideas how to fix it?

**Comment by freekmurze** [comment]
@kichetof I've opened a fix in #190. The issue is that v5.0.1 switched from `Nova::script()` to `Nova::mix()`, but the required `mix-manifest.json` file isn't present in the dist directory, so Nova can't load the field assets. The fix reverts back to `Nova::script()`.

**Reply by kichetof** [reply]
So fast! Thanks a lot!
